input/fsh/omrs-allergyIntolerance.fsh

```diff
@@ -0,0 +1,67 @@
+Profile:  OpenMRSAllergyIntolerance
+Parent:   AllergyIntolerance
+Id:       omrs-AllergyIntolerance
+Title:    "OpenMRS AllergyIntolerance"
+Description: "A FHIR Allergy as understood by OpenMRS"
+* identifier 0..1
+* code 0..1
+* category 0..*
```

Comment:
I don't see `category`, `encounter`, `onset`, `asserter`, `lastOccurance` mapped in the translator.

Reply:
oh how i missed it. thanks, Let me add them.

Reply:
Resolved them. Thanks @pmanko .

Reply:
> Let me remove them.

Hi @sherrif10  don't remove the attributes which are not supported by fhir2 just give them the cardinality of 0..0